scripts/stats/cam/exevs_href_spcoutlook_stats.sh

```diff
@@ -42,8 +42,8 @@ export maskpath=$MASKS
 export vday=$VDATE
 
 #Define the spc outlook reqions mask file path
-export COMINspcoutlook=${COMINspcoutlook:-$(compath.py -o ${NET}/${evs_ver})/prep/$COMPONENT/href}
-export SPCoutlookMask=$COMINspcoutlook/spc.$VDATE
+export EVSINspcoutlook=${EVSINspcoutlook:-$COMIN/prep/$COMPONENT/href}
```

Comment:
I feel like this one should be defined upstream in the J-job.
Line 113 of `JEVS_CAM_STATS` is
```
export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/$COMPONENT}
```
Perhaps EVSINspcoutlook and its usage can be changed to EVSINspcotlk?
Or, perhaps line 45 of this script could be changed to
```
export EVSINspcoutlook=$EVSINspcotlk/href
```